packages/workspace/src/schematics/ng-new/schema.json

```diff
@@ -92,7 +92,7 @@
         "full-stack",
         "empty"
       ],
-      "default": "empty",
+      "default": "angular",
```

Comment:
Should this have changed?

Reply:
Yup. I want it to default to angular cause it is the first option in the list.